app/views/layout.nunjucks

```diff
@@ -24,6 +24,21 @@
     <!--[if IE 8]><link href="./css/nhsuk-ie8.css" media="screen" rel="stylesheet" type="text/css" /><![endif]-->
 
     <link rel="stylesheet" href="./css/print.css" media="print" type="text/css">
+
+    {% if HOTJAR_TRACKING_ID %}
+       <!-- rational for this code being in head: https://docs.hotjar.com/v1.0/docs/manual -->
+      <script>
+        (function(h,o,t,j,a,r){
+        h.hj=h.hj||function(){(h.hj.q=h.hj.q||[]).push(arguments)};
+        h._hjSettings={hjid:{{ HOTJAR_TRACKING_ID }},hjsv:5};
```

Comment:
No it doesn't - See this https://docs.hotjar.com/v1.0/docs/hotjar-tracking-code
(which I would have seen if I've read the comment!).
Hotjar IDs are numeric or at least need to be if they are to work but that is something provided by Hotjar...